Add wlpeerj cls file

asmeurer opened the pull request:
I don't have time to work on this more now, so if someone else wants to, feel free to push to this branch. [This page](https://www.overleaf.com/latex/templates/latex-template-for-peerj-journal-and-pre-print-submissions/ptdwfrqxqzbn) has some more information.

asmeurer committed:
Add wlpeerj cls file

rouckas committed:
Add year to citation of pmint

The year is provided in pmint source, so I think that it is OK to
add it to the reference.

rouckas committed:
Switch to numbered citation in the wlpeerj.cls

We can switch to author-year citations later if needed

rouckas committed:
Modify author list generation for wlpeerj

rouckas committed:
Use the peerj template for paper and supplement

The template is not suited for supplement, so a custom title was added
to supplement.

rouckas committed:
Add TODO list for peerj implementation

rouckas committed:
Remove custom modifications from wlpeerj.cls

rouckas committed:
Add keywords

isuruf commented:
@rouckas, I've merged your branch to this branch. You can push to `peerj` branch if you want to update this PR

asmeurer committed:
Add sentence to abstract about supplement

This was the sentence that I added for the PeerJ preprint (PeerJ has a minimum
character count for abstracts, and this was required to be above that count).
